scripts/preinst.sh

```diff
@@ -0,0 +1,9 @@
+echo -n "WARNING: This package contains security vulnerabilities. Do you want to want to continue installation ? (y/n) "
```

Comment:
@umagmrit @dasiyogesh are there any known issues in using the bellow preinstall script to display a warning ?

Reply:
Uma told that it should work, the only catch is we will not get access to the files of the package in the pre-install script. but we are not dependent on that.

Reply:
Thanks for the response @silentsakky